wallet/how-to/manage-networks/use-multichain.md

```diff
@@ -0,0 +1,245 @@
+          methods: [
+            "personal_sign",
+            "eth_blockNumber",
+            "eth_gasPrice",
+            "eth_getBalance",
+            "eth_getTransactionCount",
+            "eth_sendTransaction",
+            "eth_subscribe"
+          ],
+          notifications: ["eth_subscription"],
+          accounts: []
+        },
+        "eip155:59141": { // Linea Sepolia
+          methods: [
+            "personal_sign",
+            "eth_blockNumber",
+            "eth_gasPrice",
+            "eth_getBalance",
+            "eth_getTransactionCount",
+            "eth_sendTransaction", 
+            "eth_subscribe"
+          ],
+          notifications: ["eth_subscription"],
+          accounts: []
+        }
+      }
+    }
+  }
+});
+```
+
+In `optionalScopes`:
+
+- Request access to networks that your dapp intends to interact with.
+  If a requested network is not configured by the MetaMask user, you might need to
+  [add the network](add-network.md).
+- For each network, request access to [Wallet API methods](../../reference/json-rpc-methods/index.md)
+  that your dapp expects to call at any time.
+  The methods listed in the `sessionScope` of each Multichain API response indicate which wallet
+  capabilities your dapp can use during the connection.
+
+#### 2.3. Check for connection changes
+
+To ensure your dapp responds appropriately to changes in the multichain connection, such as network or
+account updates, check for connection changes using the
+[`wallet_sessionChanged`](../../reference/multichain-api.md#wallet_sessionchanged) event.
+Based on the event data, you can determine whether your dapp needs to request additional permissions
+using [`wallet_createSession`](../../reference/multichain-api.md#wallet_createsession).
+
+```javascript
+extensionPort.onMessage.addListener((msg) => {
+  // Check for wallet_sessionChanged events.
+  if (msg.data.method === "wallet_sessionChanged") {
+    // Update permissions if required.
+  }
+});
+```
+
+### 3. Invoke Wallet API methods
+
+You can invoke a subset of the [Wallet JSON-RPC API methods](../../reference/json-rpc-methods/index.md)
+on a specified chain using the [`wallet_invokeMethod`](../../reference/multichain-api.md#wallet_invokemethod)
+Multichain API method.
+The following are example Wallet API functionalities that are compatible with the Multichain API.
+
+#### 3.1. Sign in with Ethereum
+
+You can implement Sign-In with Ethereum (SIWE) by invoking
+[`personal_sign`](/wallet/reference/json-rpc-methods/personal_sign).
+For example:
+
+```javascript
+// Specify an account that the signature will be requested for.
+const address = "0xAddress";
+const message = `Sign-in request for ${address} at ${new Date().toLocaleString()}`;
+
+// Invoke the personal_sign Wallet API method.
+const sign = await extensionPort.postMessage({
+  type: "caip-348",
+  data: {
+    "jsonrpc": "2.0",
+    method: "wallet_invokeMethod",
+    params: {
+      scope: "eip155:1",
```

Comment:
worth including the `// Ethereum Mainnet` comment here and below too? unsure how familiar our readers will be with CAIP-2 Chain ID formats